src/schemas/appData.schema.json

```diff
@@ -104,8 +104,23 @@
           },
           "version": {
             "$ref": "#/definitions/version"
+          },
+          "slippageInBips": {
+            "$id": "#/definitions/quote/slippageInBips",
+            "title": "Slippage percentage stored in Basis Points (BIPS)",
+            "examples": ["5", "10", "20", "100"],
+            "pattern": "^\\d+(\\.\\d+)?$",
+            "type": "string"
           }
-        }
+        },
+        "dependencies": {
+          "sellAmount": ["buyAmount"],
+          "buyAmount": ["sellAmount"]
+        },
```

Comment:
JSON schema trick to make `sellAmount` and `buyAmount` required when one of them is present